### PR TITLE
Fix flow types for createRPCReducer interface

### PR DESCRIPTION
**Pull request by ganemone**
This fixes issues with the `createRPCReducer` interface when working with flow types by using generics rather than the existential operator `*` whenever possible. This allows you to create a flow typed rpc reducer using the `Reducer` type helper from redux. For example:

```js
import type {Reducer} from 'redux';
import {createRPCReducer} from 'fusion-rpc-redux';

const StateType = {|
  a: string,
  b: boolean
|};

const ActionType = {|
  type: string,
  payload: {
    a: string,
    b : boolean
  }
|}

const reducer: Reducer<StateType, ActionType> = createRPCReducer('rpcId', {
  // ...
});
```

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [ignore]
 .*/node_modules/.*[^(package)]\.json$
 <PROJECT_ROOT>/dist/.*
+<PROJECT_ROOT>/src/fixtures/
```
Comment (KevinGrandon) [inline .flowconfig]: Question - Why is this needed if we're running flow in the testing suite?

Comment (ganemone) [reply .flowconfig]: The fixtures are run separately, and some of them are intended to fail. We want that to show up in tests but not when running flow on the project
